gflownet/gflownet.py

```diff
@@ -409,7 +409,7 @@ def _add_to_batch(batch, envs, actions, valids, train=True):
             for idx in range(n_empirical):
                 env = envs[idx]
                 env = env.set_state(x_tr[idx].tolist(), done=True)
-                env.n_actions = env.get_max_traj_len()
+                env.n_actions = env.get_max_traj_length()
                 envs_offline.append(env)
                 actions.append((env.eos,))
```

Comment:
Should this be `actions.append(env.eos)` as `env.eos` is already a tuple now (if I understand correctly)

Reply:
Indeed, good catch! Updated.